documentation/service-rules/ecf1/finance-users.md

```diff
@@ -0,0 +1,215 @@
+
+A breakdown of adjustments appears at the bottom of the financial statement page, and the total adjustment value is shown in the summary section at the top.
+
+Adjustments cannot be edited after the statement has been marked as paid.
+
+### To add an adjustment
+
+- Visit the relevant statement page (`output_fee` must be `true`)
+- Scroll down to **Additional adjustments**
+- Click **Add**
+- Enter the payment type and click **Continue**
+- Enter the payment amount and click **Continue**
+- Review the summary and click **Confirm and continue**
+- Choose to add another adjustment or select **No** and click **Continue**
+
+### To change an adjustment
+
+- Visit the statement page
+- Scroll to **Additional adjustments**
+- Click **Change or remove**
+- Click **Change** on the relevant adjustment
+- Update the payment type and/or amount
+- Click **Continue** and then **Confirm and continue**
+- Click **Back** to return to the statement page
+
+### To remove an adjustment
+
+- Visit the statement page
+- Scroll to **Additional adjustments**
+- Click **Change or remove**
+- Click **Remove** on the relevant adjustment
+- Confirm and click **Continue**
+- Click **Back** to return to the statement page
+
+## Contract information
+
+Contract managers can view contract values used for statement calculation by clicking the **Contract information** link at the bottom of the financial statement page.
+
+A contract is identified by the lead provider, cohort and version. Contracts are not shared by lead providers or cohort.
+Statements from the same lead provider and cohort can share the same contract (identified by `contract_version`).
+
+The information displayed depends on the cohort:
+
+### Cohorts up to 2024
+
+- Provider name
+- Recruitment target
+- Revised recruitment target (+2%)
+- Uplift target
+- Uplift amount
+- Set-up fee
+- Bands:
+  - Band name
+  - Minimum
+  - Maximum
+  - Payment per participant
+
+### Cohorts from 2025 onwards
+
+- ECT recruitment target
+- Revised ECT recruitment target (+115%)
+- ECT bands:
+  - Band name
+  - Minimum
+  - Maximum
+  - Payment per participant
+- Mentor recruitment target
+- Mentor payment per participant
+
+## Statement versioning
+
+Statements include a `contract_version` field, which determines which version of the call off contract to use in the statement calculator. This value follows **semantic versioning**, starting from `0.0.1`.
+
+When contract values are updated:
+
+- The existing contract and its bands are duplicated
+- New contract values and band values are applied
+- The contract `version` is incremented (patch version)
+- The statement’s `contract_version` is updated to match the new contract version
+
+Contracts that are no longer linked to statements (due to it being superseded by new contract version) or identified as duplicates have their `version` prefixed with `unused_` (e.g. `unused_0.0.1`) to indicate they should not be used for statement calculations.
```

Comment:
Contracts that are no longer linked to statements (due to them being superseded by a new contract version)